### ISSUE TITLE
release v2.1.0 miss header file of boost::mpl::list in postgres_backend_tests.cpp

### ISSUE THREAD

**Issue by gzliudan** [EOSIO/eos#10439]
When I compile EOS v2.1.0, below error happens:
```bash
[ 90%] Building CXX object plugins/blockvault_client_plugin/tests/CMakeFiles/blockvault_unittests.dir/postgres_backend_tests.cpp.o
/home/....../plugins/blockvault_client_plugin/tests/postgres_backend_tests.cpp:81:21: error: no template named 'list' in namespace 'boost::mpl'
typedef boost::mpl::list<eosio::blockvault::postgres_backend> test_types;
```

After I add a line: 
```cpp
#include <boost/mpl/list.hpp>
```
the above error disappears. I checked the develop branch, plugins/blockvault_client_plugin/tests/postgres_backend_tests.cpp has this include line.

**Comment by gzliudan** [comment]
Yes, I'm using boost 1.76

**Comment by gzliudan** [comment]
Maybe it's time to release v2.1.1 now.

**Comment by nksanthosh** [comment]
Thank you. The fix now implemented in #10448 will be included in the 2.1.1 release.
